calculator.go

```diff
@@ -0,0 +1,12 @@
+package Calculator
+
+func add(x int, y int) int {
```

Comment:
nit: it's probably not useful to have non-exported fns for a `calculator` class..